src/checkbox/checkbox.tsx

```diff
@@ -14,16 +15,19 @@ const Checkbox: React.FunctionComponent<Props> = (props) => {
     };
     // TODO: Upgrade to useCallback (React API Hooks)
     return (
-        <input
-            id={props.id}
-            type="checkbox"
-            checked={props.value}
-            data-qa={`checkbox-${props.id}`}
-            onChange={changeHandler}
-            disabled={props.disabled}
-            className={props.className}
-            aria-checked={props.value}
-        />
+        <>
+            <input
+                id={props.id}
+                type="checkbox"
+                checked={props.value}
+                data-qa={`checkbox-${props.id}`}
+                onChange={changeHandler}
+                disabled={props.disabled}
+                className={`checkbox ${props.className}`}
+                aria-checked={props.value}
+            />
+            <label htmlFor={props.id}>{props.label || ''}</label>
```

Comment:
Introducing the label here seems a bit weird. I'd expect these two things to be separate. Anything speaking against it? Are we already doing it somewhere like that?

Reply:
Also not tested, currently.

Reply:
It's used mainly for creating custom styled checkbox component.

Reply:
Clarified.